### DISCUSSION TITLE
Thanos receive panic error

### DISCUSSION THREAD

**Discussion by dirkckau** [thanos-io/thanos#5252] [Questions & Answers]
Hi, 

Did anyone met this error before? 
In my testing environment, there are numbers of Thanos Receivers, each of them using a individual pvc.  

One of the receiver started to has the error "panic: write header: xxxxxxx no space left on device". 
Since then, the other components start to the have the space errors. 
The retention feature is lost and finally the Thanos Receive is not functioning. 

![image](https://user-images.githubusercontent.com/91056335/159839159-098c3832-22ef-46f9-aec7-b474a9db2664.png)

**Comment by wiardvanrij** [comment]
This means that the disk is full, so that behaviour could be expected if that's the case.
